Packs/Gmail/Integrations/Gmail/Gmail_test.py

```diff
@@ -228,7 +232,259 @@ def test_labels_to_entry():
+    assert result.raw_response == expected_result.get('expected_raw_response')
+
+
+title_test_role_to_entry = 'Role 00000000000000000 details:'
+
+
+@pytest.mark.parametrize('title, role, expected_result', [
+    (title_test_role_to_entry, input_data.role_test_role_to_entry, input_data.expected_result_test_role_to_entry)
+])
+def test_role_to_entry(title, role, expected_result):
+    """
+    Tests role_to_entry function.
+        Given:
+            - gmail get role api response (from get_role function).
+        When:
+            - executing role_to_entry function.
+        Then:
+            -the outputs, raw response, and human readable are valid.
+    """
+
+    from Gmail import role_to_entry
+
+    result = role_to_entry(title, role)
+    assert result.readable_output == expected_result.get("expected_human_readable")
+    assert result.outputs == expected_result.get("expected_outputs")
+    assert result.raw_response == expected_result.get('expected_raw_response')
+
+
+title_test_user_roles_to_entry = 'User Roles of 222222222222222222222:'
+
+
+@pytest.mark.parametrize('title, response, expected_result', [
+    (title_test_user_roles_to_entry, input_data.get_user_role_mock_result,
+     input_data.expected_result_user_roles_to_entry)
+])
+def test_user_roles_to_entry(title, response, expected_result):
+    """
+    Tests user_roles_to_entry function.
+        Given:
+            - gmail get user role api response (from get_user_role function).
+        When:
+            - executing user_roles_to_entry function.
+        Then:
+            -the outputs, raw response, and human readable are valid.
+    """
+
+    from Gmail import user_roles_to_entry
+
+    result = user_roles_to_entry(title, response)
+    assert result.readable_output == expected_result.get("expected_human_readable")
+    assert result.outputs == expected_result.get("expected_outputs")
+    assert result.raw_response == expected_result.get('expected_raw_response')
+
+
+title_test_tokens_to_entry = 'Tokens:'
+
+
+@pytest.mark.parametrize('title, response, expected_result', [
+    (title_test_tokens_to_entry, input_data.get_user_tokens_mock_result,
+     input_data.expected_result_test_tokens_to_entry)
+])
+def test_tokens_to_entry(title, response, expected_result):
+    """
+    Tests tokens_to_entry function.
+        Given:
+            - gmail get user tokens api response (from get_user_tokens function).
+        When:
+            - executing tokens_to_entry function.
+        Then:
+            -the outputs, raw response, and human readable are valid.
+    """
+
+    from Gmail import tokens_to_entry
+
+    result = tokens_to_entry(title, response)
+    assert result.readable_output == expected_result.get("expected_human_readable")
+    assert result.outputs == expected_result.get("expected_outputs")
+    assert result.raw_response == expected_result.get('expected_raw_response')
+
+
+title_sent_mail_to_entry = 'Email sent:'
```

Comment:
Use this directly in the parametrize section (below)
No need to declare it here and then use it
Same goes for all